dali/operators/reader/numpy_reader_op.cc

```diff
@@ -116,6 +236,12 @@ bool NumpyReader::SetupImpl(std::vector<OutputDesc> &output_desc,
   int ndim = file_0.image.shape().sample_dim();
   TensorListShape<> sh(batch_size, ndim);
 
+  bool need_slice = slice_attr_.template ProcessArguments<CPUBackend>(ws, batch_size, ndim);
```

Comment:
`ProcessArguments` returns true if the arguments were there an were successfully parsed - but it will also return true if there were 0 axes specified (no slicing).